domain/src/main/java/com/example/domain/usecase/CharacterUseCaseImpl.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.domain.usecase
+
+import com.example.domain.entity.CharacterDataWrapperEntity
+import com.example.domain.repository.CharacterRepository
+import com.example.domain.utils.Result
+
+class CharacterUseCaseImpl(private val characterRepository: CharacterRepository) : CharacterUseCase {
```

Comment:
for the useCase you can put in the same file the interface and the implementation

Reply:
done